src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -169,6 +169,11 @@ uct_rdmacm_cm_device_context_init(uct_rdmacm_cm_device_context_t *ctx,
               log_max_num_reserved_qpn, ctx->log_reserved_qpn_granularity);
 
     ctx->use_reserved_qpn = 1;
+    ctx->reuse_qpn        = cm->config.reuse_qpn;
```

Comment:
@evgeny-leksikov For your comment ```maybe just check config everywhere?```, I follow ```ctx->use_reserve_qn``` usage in the existed code. In line ```192: ctx->reuse_qpn        = 0;```, I also assign the configuration into per ```uct_rdmacm_cm_device_context_t``` object.